src/report-export/types/report-export-service.ts

```diff
@@ -1,18 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export type ExportFormat = 'download' | 'codepen';
+export type ReportExportServiceKey = 'download' | 'codepen';
 
-export type ExportFormProps = ExportProps & { onSubmit: () => void };
+export type ReportExportFormProps = ReportExportProps & { onSubmit: () => void };
 
-export interface ExportProps {
+export type ReportExportProps = {
     html: string;
     fileName: string;
     description: string;
-}
+};
 
-export interface ReportExportService {
-    key: ExportFormat;
+export type ReportExportService = {
+    key: ReportExportServiceKey;
     displayName: string;
-    exportForm: React.ComponentType<ExportFormProps> | null;
```

Comment:
Sanity-checking that the union type removal is intentional - AFAIK this doesn't affect behavior given that we don't seem to use `strictNullChecks`. Is that right?

Reply:
That is right, but also a bigger hammer than necessary here; we only have one implementation of a service and it doesn't ever set this to null :)